android/cli/commands/_build.js

```diff
@@ -3697,6 +3697,17 @@ AndroidBuilder.prototype.generateAndroidManifest = function generateAndroidManif
 		});
 	}
 
+	if (this.realTargetSDK >= 24) {
+		Object.keys(finalAndroidManifest.application.activity).forEach(function (name) {
+			var activity = finalAndroidManifest.application.activity[name];
+			if (!activity.configChanges) {
+				activity.configChanges = ['density'];
```

Comment:
The "density" configChange should always be injected into the AndroidManifest.xml for every activity when targeting Android 7.0 or higher.

If we do not add this configChange setting, then a Titanium activity's UI will disappear if the end-user changes the "Display Size" under Android's main "Settings/Display" screen.

The reason this happens is because apps targeting Android 7.0 (or higher) are expected to handle "Display Size" changes dynamically.  Apps targeting an older OS version (ie: API Level) are force-quit and restarted instead when the "Display Size" changes.